O3exp/pipeline/analysis.ini

```diff
@@ -71,8 +71,27 @@ do-trigger-fitting =
 ; any cuts to remove super-glitchy templates are assumed to have been made in the bank
 ;background-bins = all:total:lt600
 
-[workflow-coincidence-full]
-parallelization-factor = 20
+[workflow-coincidence-full3d]
+; calculates 3-ifo coincidences in triple coincident time
+parallelization-factor = 1
+
+[workflow-coincidence-full2d0]
+; calculates H-L coincidences over all times when H1 and L1 are observing
+parallelization-factor = 1
```

Comment:
btw, why is parallelization-factor changed from 20 to 1 here, is the new code much more efficient or does it not yet handle parallelization?

Reply:
It handles it, I just wanted to create 1 file for each case while testing. I can set that back to 20.

Reply:
ok, but I think we should keep 20 as a standard value unless @ahnitz disagrees or there is a definite reason to change.